### ISSUE TITLE
Customize "No results!" message

### ISSUE THREAD

**Issue by vebjorre** [metabase/metabase#23826]
**Problem**
Several dashboards in my organization are used to show issues in different applications, e.g. legacy implementations per team. Each team strive to have zero results in such a dashboard, i.e. cards showing "No results!", which looks like a _negative_ dashboard error.

**Wanted solution**
An option to customize the message when a question returns no results, such as "Great work! Your team has no known issues with x" (fully customizable by the card creator). Defined per card/question.

**Possible extensions**
Choose text color and background color.
Select one (or no) figure from a set defined by metabase. Such as green checkmark, yellow/red "!" or some more creative ones. Honestly just being able to choose a more positive figure than the "fly-infected disk" would be good enough.

**Importance**
A simple feature that would be a game changer for dashboards tracking issues/warnings per team. A great way to motivate teams to fix their issues.
<img width="443" alt="No results card" src="https://user-images.githubusercontent.com/55921651/178218322-64164143-a34b-42f7-9beb-c11c6eb43cce.png">

**Comment by albertoperdomo** [comment]
We implemented customizatoin of images via https://github.com/metabase/metabase/issues/39246. This ships in Metabase v50.

**Comment by vebjorre** [comment]
Yes, the text message is the most important in our case. I enjoy the new customization options from v50, but I find they don't add a lot of value for us when the option is on application level only. For it to make sense for us it needs to be customizable on question level as well. It is difficult to generalize such options in an organization with 500+ dashboards.

**Comment by dragonsahead** [comment]
A customer had to do a SQL question to customize this (leaving all the rest of the functionality behind)

EDIT: the customer wants to show the columns and a "no results" message

**Comment by mazameli** [comment]
My hunch here is that making the image more neutral and simply removing the exclamation point from the text would remove the impression that no results is a negative thing (or a positive thing), and just make it a neutral thing.

Maybe we'd still want to provide customizable text, but I think we should do the above regardless.
